src/scss/typography.scss

```diff
@@ -0,0 +1,37 @@
+.header2 {
+  font-size: $header-font-size;
+  text-transform: uppercase;
+  margin-bottom: $margin;
+  vertical-align: bottom;
+  text-align: center;
+  margin-bottom: 0;
+  i.fa-plus {
+    margin-left: $inner-padding;
+    cursor: pointer;
+  }
+}
+
+.header3 {
+  font-size: 15px;
+  text-transform: uppercase;
+  padding: 10px;
+  background-color: $mblue-bg;
+  color: #fff;
+}
+
+.header4 {
```

Comment:
Feels a little odd to have these classes map so directly to specific header tags; I've had a preference for using alternative hierarchies like `primary-header`, `subheader`, etc instead of 1-4 to avoid conflation with `h1` etc